modules/renter/hostdb/hostdb.go

```diff
@@ -242,3 +254,16 @@ func (hdb *HostDB) RandomHosts(n int, excludeKeys []types.SiaPublicKey) ([]modul
 	}
 	return hdb.hostTree.SelectRandom(n, excludeKeys), nil
 }
+
+// QueuedScans returns the currently queued scans of the hostdb.
+func (hdb *HostDB) QueuedScans() (hosts []modules.HostDBEntry, err error) {
```

Comment:
Should this return the list of hosts, or just the quantity? Returning the full list seems like overkill.